moveit_ros/planning/trajectory_execution_manager/src/trajectory_execution_manager.cpp

```diff
@@ -1554,4 +1566,26 @@ void TrajectoryExecutionManager::loadControllerParams()
     }
   }
 }
+
+double TrajectoryExecutionManager::getJointAllowedStartTolerance(std::string const& jointName) const
+{
+  auto start_tolerance_it = joints_allowed_start_tolerance_.find(jointName);
+  return start_tolerance_it != joints_allowed_start_tolerance_.end() && start_tolerance_it->second > 0 ?
+             start_tolerance_it->second :
+             allowed_start_tolerance_;
```

Comment:
The value `allowed_start_tolerance_ = 0` had a special meaning: don't validate all joints. I suggest keeping this special meaning for individual joints: Zero should mean to not validate a particular joint. This requires some changes throughout this PR.